nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/service/StandardControllerServiceProvider.java

```diff
@@ -457,6 +457,7 @@ public ControllerServiceNode getControllerServiceNode(final String serviceIdenti
     public Set<String> getControllerServiceIdentifiers(final Class<? extends ControllerService> serviceType) {
         final Set<String> identifiers = new HashSet<>();
         for (final Map.Entry<String, ControllerServiceNode> entry : controllerServices.entrySet()) {
+            Class<? extends ControllerService> c = entry.getValue().getProxiedControllerService().getClass();
```

Comment:
I don't understand why this line was added. Was the intent to reference 'c' below, rather than having that expression inline?